src/Packagist/WebBundle/Resources/views/Package/viewPackage.html.twig

```diff
@@ -212,9 +316,6 @@
                 <h1>README</h1>
                 {{ package.readme|raw }}
             </div>
-            <div class="hidden readme-expander">
```

Comment:
Jordi removed it for #607 (show full readme by default) ([changeset](https://github.com/composer/packagist/commit/b7dfeac24e010f27c5b32dc117b1f763bd585122#diff-553142c8b38c82f0b6b1c75c63e9edc3L220)). It's the current state of [`master`](https://github.com/composer/packagist/blob/master/src/Packagist/WebBundle/Resources/views/Package/viewPackage.html.twig#L291).